tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/date_and_time.py

```diff
@@ -10,6 +10,7 @@
 
 
 class DateAndTime(Base):
+    _screen_locator = (By.ID, 'dateTime')
```

Comment:
I changed _page_locator to _screen_locator, basically it is used to scroll the date and time view (screen) in settings